Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -101,6 +101,10 @@ configuration:
   name: fetch_time
   required: false
   type: 0
+- display: Use Python requests handler
+  name: use_requests_handler
+  required: false
```

Comment:
don't we want it to be the requests handler by default?

Reply:
the splunk sdk is pretty fragile in regards to connection, and im concerned making this by default might break bc
i suggest we gain some experience with it and then maybe in set it to be by default